addons/arsenal/XEH_preInit.sqf

```diff
@@ -15,55 +15,61 @@ PREP_RECOMPILE_END;
 [QGVAR(statsToggle), {
     params ["_display", "_showStats"];
 
+    _showStats = GVAR(showStats) && _showStats;
```

Comment:
`GVAR(showStats)` isn't even used - I want to remove it, but was wondering if there are any objections.

Reply:
@BrettMayson do you use that at all?

Reply:
I do not